lib/Elevate/Blockers/Grub2.pm

```diff
@@ -99,6 +103,28 @@ sub _blocker_blscfg ($self) {
     return 0;
 }
 
+sub _blocker_grub_not_installed ($self) {
+
+    return 0 if Cpanel::Pkgr::is_installed('grub2-pc');
+
+    return $self->has_blocker( <<~EOS );
+    The grub2-pc package is not installed. The GRUB2 boot loader is
+    required to upgrade via leapp.
+    EOS
+}
+
+sub _blocker_grub_config_missing ($self) {
+
+    if ( !-e '/boot/grub2/grub.cfg' && !-e '/boot/grub/grub.cfg' ) {
```

Comment:
We should use `-f` instead of `-e` here.  Possibly `-s` too to make sure it is not a touch file.